harpy/common/workflow.py

```
@@ -0,0 +1,289 @@
+            print_error("snakefile missing", f"The required snakefile [blue bold]{self.snakefile}[/] was not found in the Harpy installation.")
+            print_solution("There may be an issue with your Harpy installation, which would require reinstalling Harpy. Alternatively, there may be in a issue with your conda/mamba environment or configuration.")
+            sys.exit(1)
+        with resources.as_file(source_file) as _source:
+            shutil.copy2(_source, dest_file)
+
+    def fetch_script(self, target: str) -> None:
+        """
+        Retrieve the target harpy script and write it into workdir/scripts
+        """
+        dest_dir = os.path.join(self.workflow_directory, "scripts")
+        dest_file = os.path.join(dest_dir, target)
```

Ruff (0.12.2)

Local variable `dest_file` is assigned to but never used

Remove assignment to unused variable `dest_file`

(F841)